### DISCUSSION TITLE
Welcome to OpenFusion Discussions!

### DISCUSSION THREAD

**Discussion by OpenFusionProject** [OpenFusionProject/OpenFusion#215] [General]
Hello all! As a request from some of our maintainers I've setup discussions so that maintainers outside of our discord can have a place to discuss ideas/collab in a more public way. We'll try to migrate important discussions over to be a bit more transparent in our internal discussions.

Any maintainers, please introduce yourself if you feel comfortable!

**Comment by CPunch** [comment]
Hello! I'm CPunch! I mostly work on the Lua branch right now, but I'm honestly just surprised with the progress we've made in the past 9 months. It wouldn't have ever gotten this far without the time & dedication from our maintainers. Thank you all!

**Comment by CakeLancelot** [comment]
Hey everyone! My name's Cake, and when it comes to OpenFusion I guess you could say I'm a jack of all trades. I develop the electron client, help maintain the public servers, coordinate gruntwork, and occasionally push a commit or two. 

I hope that that the discussion tab proves useful for those wanting to ask a question, post an idea, or otherwise contribute to our project. Thanks for all the support thus far!

**Comment by Eperty123** [comment]
Hey there! I'm Eperty123. I'm one of the Resource members of OpenFusion. While I don't work on the project as much as the other members do I've done my fair share of contributions in the early days, If you are interested in seeing some of the third party tools I've made for the project do check by!

**Comment by flacmedia** [comment]
Is there a way to play on mac?

**Comment by robertcabrera4871** [comment]
Quick question, I was trying to run this code on Visual Studio, and got
[WARN] Settings: missing config.ini file!
[INFO] OpenFusion v1.4-53-gf126b88
[INFO] Protocol version: 104
[INFO] Intializing Packet Managers...
[FATAL] Critical tdata file missing: xdt.json

C:\Users\RobertCabrera\Documents\GitHub\OpenFusion\bin\winfusion.exe (process 47120) exited with code 1.
To automatically close the console when debugging stops, enable Tools->Options->Debugging->Automatically close the console when debugging stops.
Press any key to close this window . . .

I'm a novice with C++ But I wanted to see if I could get it running in my IDE

**Reply by CPunch** [reply]
the current working directory is missing the `config.ini` file (amongst other things). you can fix this by setting the working directory in your [project settings](https://forum.qt.io/topic/10977/set-current-working-directory-for-visual-studio-projects)

additionally, please read [this wiki entry](https://github.com/OpenFusionProject/OpenFusion/wiki/Compilation-with-CMake-or-Visual-Studio) for our general workflow with visual studio. i personally prefer using the vs dev command prompt to make builds using cmake (which should resolve your issue if you run it while cd'd into the project root directory).

also welcome to the project :yum:

**Comment by BowtieAidan** [comment]
I have a question, is 1.3 still playable? just curious

**Reply by dongresource** [reply]
Yes, you can always run that build of the server if you want to. That version of the launcher won't work unmodified, however, because we changed where the game assets are fetched from in the 1.4 update and the old way no longer works. You can use the newer launcher with the older server just fine.

**Reply by BowtieAidan** [reply]
will there be new updates in a near future?

**Reply by dongresource** [reply]
We do still occasionally work on the project; mostly technical changes like refactoring the codebase and fixing bugs, as well as implementing the last few little details like escort mission NPCs. So there will be future releases.

If you mean in-game content updates on our public servers, then no, we don't currently have any planned.

**Reply by taylorgang420** [reply]
Why don't the nanos appear on the screen? how can i correct this error

**Comment by taylorgang420** [comment]
Why don't the nanos appear on the screen? how can i correct this error

**Reply by dongresource** [reply]
It's a known bug with the Academy client. There isn't much that can be done about it. You can close the game and reopen it a few times until they show up.

**Comment by Clayjumper** [comment]
Is there a way to get a Fusionfall Original build with no server no login an completely offline game with the open world the full game file size like 50gb
